bridge-cli/src/omni_connector_command.rs

```diff
@@ -173,6 +173,11 @@ pub enum OmniConnectorSubCommand {
         #[command(flatten)]
         config_cli: CliConfig,
     },
+    #[clap(about = "Get version of Solana OmniBridge program")]
+    SolanaGetVersion {
```

Comment:
I think we should move some chain-specific stuff under subcommands at some point